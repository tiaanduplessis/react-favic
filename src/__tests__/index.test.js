import React from 'react'
import { render, cleanup } from 'react-testing-library'

import ReactFavic from '../'

afterEach(cleanup)

test('ReactFavic should add meta headers', () => {
  const favicons = render(
    <ReactFavic name='Lesson Desk Head Office' color='#797979' href='static/images/meta' />
  )
  expect(favicons).toMatchSnapshot()
})
