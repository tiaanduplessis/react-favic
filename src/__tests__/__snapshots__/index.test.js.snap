// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReactFavic should add meta headers 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <link
        href="static/images/meta/apple-touch-icon-57x57.png"
        rel="apple-touch-icon-precomposed"
        sizes="57x57"
      />
      <link
        href="static/images/meta/apple-touch-icon-60x60.png"
        rel="apple-touch-icon-precomposed"
        sizes="60x60"
      />
      <link
        href="static/images/meta/apple-touch-icon-72x72.png"
        rel="apple-touch-icon-precomposed"
        sizes="72x72"
      />
      <link
        href="static/images/meta/apple-touch-icon-76x76.png"
        rel="apple-touch-icon-precomposed"
        sizes="76x76"
      />
      <link
        href="static/images/meta/apple-touch-icon-114x114.png"
        rel="apple-touch-icon-precomposed"
        sizes="114x114"
      />
      <link
        href="static/images/meta/apple-touch-icon-120x120.png"
        rel="apple-touch-icon-precomposed"
        sizes="120x120"
      />
      <link
        href="static/images/meta/apple-touch-icon-144x144.png"
        rel="apple-touch-icon-precomposed"
        sizes="144x144"
      />
      <link
        href="static/images/meta/apple-touch-icon-152x152.png"
        rel="apple-touch-icon-precomposed"
        sizes="152x152"
      />
      <link
        href="static/images/meta/favicon-16x16.png"
        rel="icon"
        sizes="16x16"
        type="image/png"
      />
      <link
        href="static/images/meta/favicon-32x32.png"
        rel="icon"
        sizes="32x32"
        type="image/png"
      />
      <link
        href="static/images/meta/favicon-96x96.png"
        rel="icon"
        sizes="96x96"
        type="image/png"
      />
      <link
        href="static/images/meta/favicon-128.png"
        rel="icon"
        sizes="128x128"
        type="image/png"
      />
      <link
        href="static/images/meta/favicon-128x128.png"
        rel="icon"
        sizes="128x128"
        type="image/png"
      />
      <link
        href="static/images/meta/favicon-196x196.png"
        rel="icon"
        sizes="196x196"
        type="image/png"
      />
      <meta
        content="Lesson Desk Head Office"
        name="application-name"
      />
      <meta
        content="#797979"
        name="msapplication-TileColor"
      />
      <meta
        content="static/images/meta/mstile-144x144.png"
        name="msapplication-TileImage"
      />
      <meta
        content="static/images/meta/mstile-70x70.png"
        name="msapplication-square70x70logo"
      />
      <meta
        content="static/images/meta/mstile-150x150.png"
        name="msapplication-square150x150logo"
      />
      <meta
        content="static/images/meta/mstile-310x150.png"
        name="msapplication-wide310x150logo"
      />
      <meta
        content="static/images/meta/mstile-310x310.png"
        name="msapplication-square310x310logo"
      />
    </div>
  </body>,
  "container": <div>
    <link
      href="static/images/meta/apple-touch-icon-57x57.png"
      rel="apple-touch-icon-precomposed"
      sizes="57x57"
    />
    <link
      href="static/images/meta/apple-touch-icon-60x60.png"
      rel="apple-touch-icon-precomposed"
      sizes="60x60"
    />
    <link
      href="static/images/meta/apple-touch-icon-72x72.png"
      rel="apple-touch-icon-precomposed"
      sizes="72x72"
    />
    <link
      href="static/images/meta/apple-touch-icon-76x76.png"
      rel="apple-touch-icon-precomposed"
      sizes="76x76"
    />
    <link
      href="static/images/meta/apple-touch-icon-114x114.png"
      rel="apple-touch-icon-precomposed"
      sizes="114x114"
    />
    <link
      href="static/images/meta/apple-touch-icon-120x120.png"
      rel="apple-touch-icon-precomposed"
      sizes="120x120"
    />
    <link
      href="static/images/meta/apple-touch-icon-144x144.png"
      rel="apple-touch-icon-precomposed"
      sizes="144x144"
    />
    <link
      href="static/images/meta/apple-touch-icon-152x152.png"
      rel="apple-touch-icon-precomposed"
      sizes="152x152"
    />
    <link
      href="static/images/meta/favicon-16x16.png"
      rel="icon"
      sizes="16x16"
      type="image/png"
    />
    <link
      href="static/images/meta/favicon-32x32.png"
      rel="icon"
      sizes="32x32"
      type="image/png"
    />
    <link
      href="static/images/meta/favicon-96x96.png"
      rel="icon"
      sizes="96x96"
      type="image/png"
    />
    <link
      href="static/images/meta/favicon-128.png"
      rel="icon"
      sizes="128x128"
      type="image/png"
    />
    <link
      href="static/images/meta/favicon-128x128.png"
      rel="icon"
      sizes="128x128"
      type="image/png"
    />
    <link
      href="static/images/meta/favicon-196x196.png"
      rel="icon"
      sizes="196x196"
      type="image/png"
    />
    <meta
      content="Lesson Desk Head Office"
      name="application-name"
    />
    <meta
      content="#797979"
      name="msapplication-TileColor"
    />
    <meta
      content="static/images/meta/mstile-144x144.png"
      name="msapplication-TileImage"
    />
    <meta
      content="static/images/meta/mstile-70x70.png"
      name="msapplication-square70x70logo"
    />
    <meta
      content="static/images/meta/mstile-150x150.png"
      name="msapplication-square150x150logo"
    />
    <meta
      content="static/images/meta/mstile-310x150.png"
      name="msapplication-wide310x150logo"
    />
    <meta
      content="static/images/meta/mstile-310x310.png"
      name="msapplication-square310x310logo"
    />
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
